index.js

```diff
@@ -0,0 +1,94 @@
+import UI from './modules/classUI.js';
+import Book from './modules/classBOOK.js';
+import { DateTime } from './luxon.js';
+
+const librarySection = document.getElementById('library');
+const formSection = document.getElementById('newbook');
+const contactSection = document.getElementById('contact');
+const libraryLink = document.getElementById('librarylink');
+const newBookLink = document.getElementById('newbooklink');
+const contactLink = document.getElementById('contactlink');
+const time = document.getElementById('time');
+
+formSection.style.display = 'none';
+contactSection.style.display = 'none';
+
+function Form() {
```

Comment:
- [ ] I will encourage you to change all functions to the arrow function so that it would help javascript in the runtime and avoid hoisting.